README.md

```diff
@@ -3871,3 +3871,9 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 We encourage you to fork this guide and change the rules to fit your team’s style guide. Below, you may list some amendments to the style guide. This allows you to periodically update your style guide without having to deal with merge conflicts.
 
 # };
+
+
+## Additional Links and References for Learning more about Javascript
+
+1. https://www.codecademy.com/learn/learn-javascript
```

Comment:
We already have a section for this kind of info: https://github.com/airbnb/javascript#resources

Can you put it there?